cfgov/privacy/forms.py

```diff
@@ -154,7 +154,7 @@ def email_body(self):
         return loader.render_to_string(self.email_template, data)
 
     def format_subject(self):
-        name = self.cleaned_data["requestor_name"]
+        name = self.cleaned_data.get("requestor_name", "")
```

Comment:
In this case I believe we should always have `requestor_name` because it is a required field.

```suggestion
        name = self.cleaned_data["requestor_name"]
```